### ISSUE TITLE
add Kruschke's original (humble teacher) loss function

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#4]
This is the loss function defined in equations (4a) and (4b) from the original ALCOVE paper, which combines the HingeLoss function in the code with a mean squared error term.

To implement this in PyTorch, see the `HingeLoss` function by Brenden as a starting point, from which you'll then want to compute the difference between the "humble teacher" values with the actual output from the model, and take the squared sum of the error.

**Comment by alexatartaglini** [comment]
Done, but could use some editing.
